gtsfm/runner/run_scene_optimizer_astronet.py

```diff
@@ -0,0 +1,102 @@
+"""Runs GTSfM on an AstroNet dataset.
+
+Author: Travis Driver
+"""
+
+import argparse
+from pathlib import Path
+
+import hydra
+from dask.distributed import Client, LocalCluster, performance_report
+from hydra.utils import instantiate
+
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.gtsfm_data import GtsfmData
+from gtsfm.loader.astronet_loader import AstroNetLoader
+from gtsfm.scene_optimizer import SceneOptimizer
+
+DATA_ROOT = Path(__file__).resolve().parent.parent.parent / "tests" / "data"
+
+logger = logger_utils.get_logger()
+
+
+def run_scene_optimizer() -> None:
```

Comment:
can we follow the same arrangement of code as other runners?

Reply:
The other runners cause a mypy error:

```
Redefining name 'args' from outer scope (line 90)pylint(redefined-outer-name)
```